en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/P/Prismatic_Wall.md

```diff
@@ -47,12 +66,13 @@ This layer is destroyed by 25 fire damage.
 \
 **Indigo:**
 The creature is [restrained](#Conditions_restrained) and makes a Constitution saving throw at the end of each of its turns.
-Once it accumulates three failed saves it permanently turns to stone, or when it accumulates three successful saves the effect ends.
+Once it accumulates three failed saves it permanently turns to stone, and is subjected to the [petrified](#Conditions_petrified) condition.
```

Comment:
WotC specifies the _petrified_ condition here, A5e doesn't. I think it makes sense to have it specified. Do you agree?